pkg/api/networkservice/mechanisms/common/constants.go

```diff
@@ -46,6 +48,10 @@ const (
 	// PCIAddressKey - PCI address of the device for the SR-IOV supported mechanisms
 	PCIAddressKey = "pciAddress"
 
+	// SRIOVTokenIDKey - Client/Endpoint SR-IOV token ID
+	// nolint:gosec
+	SRIOVTokenIDKey = "sriovTokenID"
```

Comment:
Would it make sense to name this something more generic like 'DeviceToken' ... it seems like the trick you are using here could be used more broadly than SRIOV... do I have that right?

Reply:
We actually can do this, but it can possibly lead to the following behavior if there are not only SR-IOV tokens:
1. Client requests kernel interface with `AAA` device token.
2. Forwarder tries this token as a SR-IOV token - fails.
3. Fails back to mechanism selection.
4. Forwarder tries this token as a ... token - fails.
5. ...
6. Forwarder establishes the connection.
7. Client makes refresh request.
8. Forwarder tries this token as a SR-IOV token - fails.
9. ...

So we can make a `DeviceToken` field instead of `SRIOVDeviceToken` one, but we may possibly need to add some `IsDeviceTokenSRIOV` in such case to avoid this confusion.

Reply:
Changed to `TokenIDKey`.